Source/FieldSolver/SpectralSolver/SpectralAlgorithms/PsatdAlgorithmRZ.cpp

```diff
@@ -287,6 +387,27 @@ void PsatdAlgorithmRZ::InitializeSpectralCoefficients (SpectralFieldDataRZ const
                 X2(i,j,k,mode) = c*c * dt*dt / (6._rt*ep0);
                 X3(i,j,k,mode) = - c*c * dt*dt / (3._rt*ep0);
             }
+
+            if (time_averaging && J_linear_in_time)
+            {
+                constexpr amrex::Real c2 = PhysConst::c;
+                const amrex::Real dt3 = dt * dt * dt;
+                const amrex::Real om  = c * k_norm;
+                const amrex::Real om2 = om * om;
+                const amrex::Real om4 = om2 * om2;
+
+                if (om != 0)
```

Comment:
Since `om` is a Real: is there a chance that `k_norm` is not exactly zero and we would need to do an epsilon comparison to zero here?

Otherwise for clarity:
```suggestion
                if (om != 0.0_rt)
```

Reply:
I don't think that there will be an issue since `knorm` is only zero if `kr` and `kz` are zero.

Reply:
Agree with Dave, I don't think we need a comparison at machine precision. I added the `_rt` suffix for better clarity, though.